src/app/[locale]/specs/permission/en.mdx

```diff
@@ -0,0 +1,347 @@
+export const metadata = {
+  title: 'Permissions',
+  description:
+    'Auth Permissions for Account Resources.',
+}
+
+# Permissions
+
+Authorization to atproto resources is described and granted using "permissions". A group of a permissions related to a specific Lexicon namespace (record types and API endpoints) can be bundled together as a "permission set". Both are used in the context of [OAuth](/specs/oauth) to grant client software access to account resources on a PDS. For example, the ability to write records of specific types to the user's public repository, or make authenticated API requests to remote services. Developers declare the permissions their app requires to function, and end users are shown the permissions when granting access to the app.
+
+Each resource type has a defined set of parameters that can attenuate the permission. A permission can be represented in a string format (for direct use as an OAuth scope), or as a JSON object (for use in permission sets). Permission sets are published as public lexicon schemas, and are limited to lexicon resources under the same NSID hierarchy.
+
+## Resource Types
+
+Permissions relate to account resources on PDS instances:
+
+- `repo`: Public Repository (records and collections)
+- `rpc`: Service Authentication (API calls to external services)
+- `blob`: uploaded media files
+- `identity`: DID and handle
+- `account`: hosting status, email address
+
+There is an additional "meta" resource, `include`, which is used to reference a permission set (see Permission Sets section below).
+
+This section describes these resources in more depth, including all the parameters that can constrain the permission. For each resource, one parameter may be marked as "positional", which impacts the string representation syntax (described in a later section).
+
+Resources and permissions are described in the abstract, not tied to specific PDS API endpoints. For example, `rpc` permissions describe the ability to make remote service API calls using inter-service authentication. This includes both proxied PDS requests (using HTTP headers) and fetching service auth tokens using `com.atproto.server.getServiceAuth`.
+
+### `repo`
+
+Write access to records in the account's public repository. Can be limited to specific record types (collections) or actions (eg update vs delete).
+
+Parameters:
+
+- `collection` (string array, required, positional): NSID of record types. Wildcard (`*`) is allowed and gives access to all records. Partial wildcards are not supported (eg, can not use `com.example.*`)
```

Comment:
```suggestion
- `collection` (string array, required, positional): NSID of record types. Wildcard (`*`) is allowed (when the permission is encoded as an oauth scope parameter i.e. `repo:*`) and gives access to all records. Partial wildcards are not supported (eg, can not use `com.example.*`)
```

Reply:
We should what's meant by the parenthetical more explicitly^^